runtime/doc/lsp.txt

```diff
@@ -1256,9 +1256,11 @@ format({options})                                       *vim.lsp.buf.format()*
                      name (client.name) matching this field.
                    • range (table|nil) Range to format. Table must contain
                      `start` and `end` keys with {row,col} tuples using (1,0)
-                     indexing. Defaults to current selection in visual mode
-                     Defaults to `nil` in other modes, formatting the full
-                     buffer
+                     indexing. Can also be a list of tables that contain
+                     `start` and `end` keys as described above, in which case
+                     textDocument/rangesFormatting support is required.
```

Comment:
```suggestion
                     `textDocument/rangesFormatting` support is required.
```

Reply:
Sorry if I'm missing something, but this was marked as resolved yet I'm not seeing the change...?

It's okay if you decided to ignore the suggestion, but at least tell me why :)

Reply:
Ahrg sorry, I changed it locally in lsp.txt, but then running `make doc` overwrote it again - I changed in the right place this time (buf.lua)